src/main/java/edu/cornell/kfs/fp/CuFPConstants.java

```diff
@@ -157,17 +157,16 @@ public static ScheduleTypes fromName(String name) {
         public static final String AWS_CHART_CODE_PROPERTY_NAME = "AWS_CHART_CODE";
         public static final String AWS_DEFAULT_DOCUMENT_DESCRIPTION_PROPERTY_NAME = "AWS_DEFAULT_DOCUMENT_DESCRIPTION";
         public static final String AWS_NOTE_HELP_TEXT_PROERTY_NAME = "AWS_NOTE_HELP_TEXT";
```

Comment:
Minor typo in name of constant. (This constant is being used by the new code in this PR.)